src/components/TopBar.js

```diff
@@ -187,16 +186,14 @@ const TopBar = ({ showMenu = true, title, href, children }) => {
               style: { flex: 'none', display: 'flex', justifyContent: 'center', alignItems: 'center', height: 95 },
               role: 'listitem'
             }, [
-              isDatastage() || isBioDataCatalyst() ?
-                h(Clickable, {
-                  href: Nav.getLink('workspaces'),
-                  style: {
-                    backgroundColor: 'white', fontSize: 18, fontWeight: 500, color: colors.accent(),
-                    borderRadius: 5, boxShadow: '0 2px 4px 0 rgba(0,0,0,.25)',
-                    width: 250, height: 56, display: 'flex', alignItems: 'center', justifyContent: 'center'
-                  }
-                }, ['SIGN IN']) :
-                h(SignInButton)
+              h(Clickable, {
+                ...(isDatastage() || isBioDataCatalyst()) ? { href: Nav.getLink('workspaces') } : { onClick: signIn },
+                style: {
+                  backgroundColor: 'white', fontSize: 18, fontWeight: 500, color: colors.accent(),
+                  borderRadius: 5, boxShadow: '0 2px 4px 0 rgba(0,0,0,.25)',
+                  width: 250, height: 56, display: 'flex', alignItems: 'center', justifyContent: 'center'
+                }
+              }, ['LOG IN'])
```

Comment:
Ah wait, are we using LOG IN?  I just changed it to SIGN IN in TDR 🤦 I'll change it back

Reply:
Using 'Log in' per Lou 🤷

Reply:
And NOT "LOGIN", which is a noun.